src/main/java/duke/exception/InvalidInputException.java

```diff
@@ -0,0 +1,14 @@
+package duke.exception;
+
+public class InvalidInputException extends Exception {
```

Comment:
Perhaps you can explain more on what causes this error. Where will the invalid input be from?